binhollc/SupernovaController
# BMC2-1377 | Change name of hot_join_example.py to i3c_hot_join_example.py

# Resolves  
https://focusuy.atlassian.net/browse/BMC2-1377

# How to test  
Create a virtual env `python venv venv`
install sncontroller locally ( `pip install ../path/to/sncontroller/repo` )
Check your venvs `Lib/site-packages/supernovacontrollerexamples`

# What to expect  
i3c_hot_join_example.py is included in the examples

# Notes  
Feel free to try the example, but it is not necessary
